mandatory/2-function-creation.js

```diff
@@ -5,7 +5,22 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
```

Comment:
# Hey there I just have a code refactoring tip for you 

So with this **tidyUpString function** I do not know if it passes the test cases, but however even if it does I just want to say I see an overuse of functions **unnecessary functions** 

#### For example I do not see a need for creating a function for trim meanwhile ```.trim()``` is function on it's own
```diff
- let removeSpaces = (strArr) => strArr.trim();
```

### My advise

- Try an explore as to how you can chain this 3 methods **Which you created individual functions for**
- Below this what I mean
```js
.replace(/\//gi, '').trim().toLowerCase()
```

## Hint
- Key would be to have each individual string from array in front of the ```.replace()``` method and the rest of the chained methods should now apply i.e ```.trim()``` & ```.toLowerCase()```

# Hope this helps 🔥💪

Reply:
Thank you Ntshembo.
I created the pull request after completing the exercises.
I'm still working on the mandatory folder.